apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/source/AbstractClaimTransformer.java

```diff
@@ -0,0 +1,60 @@
+package gov.cms.bfd.pipeline.rda.grpc.source;
+
+import gov.cms.bfd.model.rda.RdaClaimMessageMetaData;
+import gov.cms.bfd.pipeline.rda.grpc.RdaChange;
+import gov.cms.mpsm.rda.v1.RecordSource;
+import java.util.Map;
+
+/** Base class for Claim transformation offering common logic shared between FISS and MCS */
+public abstract class AbstractClaimTransformer {
+
+  private static final Map<String, Short> PHASE_TO_SHORT =
+      Map.of(
+          "p1", (short) 1,
+          "p2", (short) 2,
+          "p3", (short) 3);
+
```

Comment:
The short values should be constants so the tests can reference them.

Reply:
The other side of that is that tests are meant to be a challenge against business logic.  Say you were to make a `public final int PHASE_ONE = 1`, and then use that in a test.  If someone were to then change the value of `PHASE_ONE` to `0`, the test would still pass because it affects both business logic and the test.  Keeping isolated values in tests provides an extra check.